### PR TITLE
Chapter 10 Author Review

### PR DESCRIPTION
**Pull request by szimmer**

### CODE REVIEW

File: 10-ambarom-vignette.Rmd [hunk 1]
```diff
@@ -11,7 +13,7 @@ The survey has a core set of questions across the countries, but not all questio
 Each country and each year has its own files. The data used in this vignette can be downloaded from the LAPOP website.  In this vignette, we will be using data from 2021, namely version v1.2. These are not available on the book's repository, but you may download the raw files yourself^[http://datasets.americasbarometer.org/database/index.php] (@lapopdat). To read all files into R and ignore the Stata labels, we recommend running code like this:
 
 ```r
-stata_files <- list.files(here("RawData", "LAPOP_2021"), "*.dta")
+stata_files <- list.files(here("RawData", "LAPOP_2021"), "*.dta")
```
Comment (rpowell22) [inline 10-ambarom-vignette.Rmd:16]: do people need haven for this?

File: 10-ambarom-vignette.Rmd [hunk 2]
```diff
@@ -34,7 +36,7 @@ Many of the variables are coded as numeric and do not have intuitive variable na
 library(tidyverse)
 library(srvyr)
 library(sf)
-library(rnaturalearth) # Getting world maps
+library(rnaturalearth) # Getting world maps
```
Comment (rpowell22) [inline 10-ambarom-vignette.Rmd:39]: FYI the warning message when loading this package is off the page in the PDF.

File: 10-ambarom-vignette.Rmd [hunk 3]
```diff
@@ -47,7 +49,7 @@ library(ggpattern)
 library(osfr)
 source("helper-fun/helper-functions.R")
 
-ambarom_in <- read_rds_tsr("lapop_2021.rds")
+ambarom_in <- read_rds_tsr("lapop_2021.rds")
```
Comment (rpowell22) [inline 10-ambarom-vignette.Rmd:52]: we mention above that the data is not available and they have to download it themselves.  should we mask this somehow then?

File: 10-ambarom-vignette.Rmd [hunk 4]
```diff
@@ -203,7 +205,7 @@ ambarom %>% count(NewsFrequency, gi0n)
...
+The technical report is the best source to understand how to specify the sampling design in R^[https://www.vanderbilt.edu/lapop/ab2021/AB2021-Technical-Report-v1.0-FINAL-eng-030722.pdf]. The data includes two weights: `wt` and `weight1500`. The first weight variable is country-specific and sums to the sample size but is calibrated to reflect each country's demographics, while the second weight variable sums to 1500 for each country. The second weight is indicated as the weight to use for multi-country analyses. While the documentation does not directly state this, the example Stata syntax `svyset upm [pw=weight1500], strata(strata)` indicates the variable `upm` is a clustering variable, and `strata` is the strata variable. The design object is setup as follows:
```
Comment (rpowell22) [inline 10-ambarom-vignette.Rmd:208]: you call `upm` a clustering variable, but everywhere else in the book it is "PSU" change to match here?

File: 10-ambarom-vignette.Rmd [hunk 5]
```diff
@@ -246,13 +248,13 @@ Another question asked how education was affected by the pandemic. This question
...
->  - Yes, they cut all ties with the school
+>    - No, because they are not yet school age or because they do not attend school for another reason
+>    - No, their classes continued normally
+>    - Yes, they went to virtual or remote classes
+>    - Yes, they switched to a combination of virtual and in-person classes
+>    - Yes, they cut all ties with the school
 
-Multiple-choice questions are interesting. If we want to look at how education was impacted only among those in school, we need to filter to the relevant responses, which is anyone that responded no to the first part. An unweighted cross-tab for the responses is included below, and we can see there is a wide-range of impacts and that many combinations of effects on education are possible.
+Multiple-choice questions are interesting. If we want to look at how education was impacted only among those in school, we need to filter to the relevant responses, which is anyone that responded no to the first part. An unweighted cross-tab for the responses is included below, and we can see there is a wide-range of impacts and that many combinations of effects on education are possible.
```
Comment (rpowell22) [inline 10-ambarom-vignette.Rmd:257]:
```suggestion
Multiple-choice questions are interesting. If we want to look at how education was impacted only among those in school, we need to filter to the relevant responses, which is anyone that did not select the first option. An unweighted cross-tab for the responses is included below, and we can see there is a wide-range of impacts and that many combinations of effects on education are possible. 
```

File: 10-ambarom-vignette.Rmd [hunk 6]
```diff
@@ -305,14 +307,14 @@ covid_educ_ests %>%
               columns = c("p_mediumchange", "p_mediumchange_se")) %>%
   tab_spanner(label = "Cut ties with school",
               columns = c("p_noschool", "p_noschool_se")) %>%
-  fmt_number(decimals = 1)
+  fmt_number(decimals = 1)
```
Comment (rpowell22) [inline 10-ambarom-vignette.Rmd:310]: FYI the table is off the page in PDF.

**Review by rpowell22 — CHANGES_REQUESTED** [review]

File: 10-ambarom-vignette.Rmd (hunk 6 above)
Comment (ivelasq) [reply 10-ambarom-vignette.Rmd:310]: I don't know how to fix this. It looks like `tab_style()` edits to gt tables don't show up in PDF?

* https://github.com/rstudio/gt/issues/119
* https://github.com/rstudio/gt/issues/1363

**Pull request merged**